docs/UserGuide.md

```diff
@@ -359,20 +359,51 @@ Find is case-insensitive, and the order in which the keywords are entered is irr
 
 ---
 
-### Sorting the shows
-[TO CHANGE AFTER REWORK]
+### Sorting the shows `sort`
 
-Sort shows based on the input prefix.
+Your show list is in mess! Do you need a way to organise the list?
+Sort can help reorder the list!!
 
-Format: `sort [sna/] [snd/] [ssa/] [ssd/] [so/]…​`
-* Use sna/ to sort name in ascending order.
-* Use snd/ to sort name in descending order.
-* Use ssa/ to sort status in ascending order.
-* Use ssd/ to sort status in descending order.
-* If there are no prefix, it will sort by name in ascending order.
-* If both prefixes for ascending and descending are used for the same attribute, it will only sort by ascending.
-* If both prefixes for name and status are used, by default, it will sort by name then status.
-* Use so/ to sort by status then name.
+:exclamation:**Caution:**<br>
+*You cannot undo the sort once you applied it!
```

Comment:
Minor grammatical error here
Should either be `once you have applied it!` or `once you apply it!`

Reply:
ok